contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -85,6 +85,8 @@ namespace eosiosystem {
    static constexpr int64_t  default_inflation_pay_factor  = 50000;   // producers pay share = 10000 / 50000 = 20% of the inflation
    static constexpr int64_t  default_votepay_factor        = 40000;   // per-block pay share = 10000 / 40000 = 25% of the producer pay
 
+   static const     uint64_t STANDBY_PAY_RATIO_DENOMINATOR = 10000;   // base for standby pay ratio
```

Comment:
use the same coding standard with other static variables :   static constexpr int64_t  default_votepay_factor

Reply:
yes agreed